src/runtime/config/configuration-qemu.toml.in

```diff
@@ -76,6 +76,13 @@ firmware_volume = "@FIRMWAREVOLUMEPATH@"
 # For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
 machine_accelerators="@MACHINEACCELERATORS@"
 
+# Qemu seccomp sandbox feature
+# comma-separated list of seccomp sandbox features to control the syscall access.
+# For example, `seccompsandbox= "on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny"`
+# Please note, enabling this feature may reduce performance, you may enable
+# /proc/sys/net/core/bpf_jit_enable to reduce the impact. see https://man7.org/linux/man-pages/man8/bpfc.8.html
```

Comment:
@fengwang666 It will be useful to mention that this needs to be enabled inside the Kata guest. A user may try to enable this on the host side. We have a howto guide to enable sysctls with Kata: https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-use-sysctls-with-kata.md.
Will be useful to link it here.

Reply:
My bad here @fengwang666. This would be on the host side itself.

Reply:
`elevateprivileges=deny` does not work with demonize which we use.
https://gitlab.com/qemu-project/qemu/-/issues/798

> This is pretty much by design - the QEMU seccomp filters are explicitly intended to break/block certain QEMU functionality.

Reply:
@mcastelino That's an important point, thanks! This should be pointed out in the docs and `elevateprivileges=deny` should be dropped from the default value IMHO.